### PR TITLE
[linstor] Mark the module as deprecated

### PR DESCRIPTION
**Pull request by z9r5**
## Description
- Add the deprecated status for a module.
- Mark the linstor module as deprecated in the documentation

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries
```changes
section: docs
type: chore
summary: |
  - Add the deprecated status for a module.
  - Mark the linstor module as deprecated in the documentation
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: modules/041-linstor/docs/CONFIGURATION.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ force_searchable: true
 description: Settings of the linstor Deckhouse module.
 ---
 
+{% alert level="danger" %}
+The current version of the module is outdated and is no longer supported. Switch to using the [sds-drbd]() module.
```
Comment (z9r5) [inline modules/041-linstor/docs/CONFIGURATION.md:8]:
```suggestion
The current version of the module is outdated and is no longer supported. Switch to using the [sds-drbd](https://deckhouse.io/modules/sds-drbd/beta/) module.
```

File: modules/041-linstor/docs/CONFIGURATION_RU.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ force_searchable: true
 description: Параметры настройки модуля linstor Deckhouse.
 ---
 
+{% alert level="danger" %}
+Текущая версия модуля устарела и больше не поддерживается. Переключитесь на использование модуля [sds-drbd]().
```
Comment (z9r5) [inline modules/041-linstor/docs/CONFIGURATION_RU.md:8]:
```suggestion
Текущая версия модуля устарела и больше не поддерживается. Переключитесь на использование модуля [sds-drbd](https://deckhouse.ru/modules/sds-drbd/beta/).
```

File: modules/041-linstor/docs/ADVANCED_USAGE.md [hunk 3]
```diff
@@ -3,6 +3,10 @@ title: "The linstor module: advanced configuration"
 description: Steps for manually creating storage pools and StorageClasses when enabling the linstor Deckhouse module. Configuring backup snapshots of linstor volumes to S3.
 ---
 
+{% alert level="danger" %}
+The current version of the module is outdated and is no longer supported. Switch to using the [sds-drbd]() module.
```
Comment (z9r5) [inline modules/041-linstor/docs/ADVANCED_USAGE.md:7]:
```suggestion
The current version of the module is outdated and is no longer supported. Switch to using the [sds-drbd](https://deckhouse.io/modules/sds-drbd/beta/) module.
```

File: modules/041-linstor/docs/ADVANCED_USAGE_RU.md [hunk 4]
```diff
@@ -3,6 +3,10 @@ title: "Модуль linstor: расширенная конфигурация"
 description: Шаги по ручному созданию пулов хранения и StorageClass’ов при включении модуля linstor Deckhouse. Настройка резервного копирования снапшотов томов linstor в S3.
 ---
 
+{% alert level="danger" %}
+Текущая версия модуля устарела и больше не поддерживается. Переключитесь на использование модуля [sds-drbd]().
```
Comment (z9r5) [inline modules/041-linstor/docs/ADVANCED_USAGE_RU.md:7]:
```suggestion
Текущая версия модуля устарела и больше не поддерживается. Переключитесь на использование модуля [sds-drbd](https://deckhouse.io/modules/sds-drbd/beta/).
```

File: modules/041-linstor/docs/FAQ.md [hunk 5]
```diff
@@ -3,6 +3,10 @@ title: "The linstor module: FAQ"
 description: LINSTOR Troubleshooting. What is difference between LVM and LVMThin? LINSTOR performance and reliability notes, comparison to Ceph. How to add existing LINSTOR LVM or LVMThin pool. How to configure Prometheus to use LINSTOR for storing data.
 ---
 
+{% alert level="danger" %}
+The current version of the module is outdated and is no longer supported. Switch to using the [sds-drbd]() module.
```
Comment (z9r5) [inline modules/041-linstor/docs/FAQ.md:7]:
```suggestion
The current version of the module is outdated and is no longer supported. Switch to using the [sds-drbd](https://deckhouse.ru/modules/sds-drbd/beta/) module.
```

File: modules/041-linstor/docs/FAQ_RU.md [hunk 6]
```diff
@@ -3,6 +3,10 @@ title: "Модуль linstor: FAQ"
 description: Диагностика проблем LINSTOR. Когда следует использовать LVM, а когда LVMThin? Производительность и надежность LINSTOR, сравнение с Ceph. Как добавить существующий LVM- или LVMThin-пул в LINSTOR. Как настроить Prometheus на использование хранилища LINSTOR.
 ---
 
+{% alert level="danger" %}
+Текущая версия модуля устарела и больше не поддерживается. Переключитесь на использование модуля [sds-drbd]().
```
Comment (z9r5) [inline modules/041-linstor/docs/FAQ_RU.md:7]:
```suggestion
Текущая версия модуля устарела и больше не поддерживается. Переключитесь на использование модуля [sds-drbd](https://deckhouse.ru/modules/sds-drbd/beta/).
```

File: modules/041-linstor/docs/README.md [hunk 7]
```diff
@@ -3,6 +3,10 @@ title: "The linstor module"
 description: Deckhouse uses the linstor module to manage a replicated block storage in the Kubernetes cluster.
 ---
 
+{% alert level="danger" %}
+The current version of the module is outdated and is no longer supported. Switch to using the [sds-drbd]() module.
```
Comment (z9r5) [inline modules/041-linstor/docs/README.md:7]:
```suggestion
The current version of the module is outdated and is no longer supported. Switch to using the [sds-drbd](https://deckhouse.io/modules/sds-drbd/beta/) module.
```

File: modules/041-linstor/docs/README_RU.md [hunk 8]
```diff
@@ -3,6 +3,10 @@ title: "Модуль linstor"
 description: Deckhouse с помощью модуля linstor управляет в кластере Kubernetes реплицируемым блочным хранилищем.
 ---
 
+{% alert level="danger" %}
+Текущая версия модуля устарела и больше не поддерживается. Переключитесь на использование модуля [sds-drbd]().
```
Comment (z9r5) [inline modules/041-linstor/docs/README_RU.md:7]:
```suggestion
Текущая версия модуля устарела и больше не поддерживается. Переключитесь на использование модуля [sds-drbd](https://deckhouse.ru/modules/sds-drbd/beta/).
```

File: modules/041-linstor/docs/USAGE.md [hunk 9]
```diff
@@ -3,6 +3,10 @@ title: "The linstor module: configuration examples"
 description: Example of configuring the linstor Deckhouse module. LINSTOR storage configuration. Placing the application near the data (data locality). Migration of the application in case of problems with the node (storage-based fencing).
 ---
 
+{% alert level="danger" %}
+The current version of the module is outdated and is no longer supported. Switch to using the [sds-drbd]() module.
```
Comment (z9r5) [inline modules/041-linstor/docs/USAGE.md:7]:
```suggestion
The current version of the module is outdated and is no longer supported. Switch to using the [sds-drbd](https://deckhouse.io/modules/sds-drbd/beta/) module.
```

File: modules/041-linstor/docs/USAGE_RU.md [hunk 10]
```diff
@@ -3,6 +3,10 @@ title: "Модуль linstor: примеры конфигурации"
 description: Примеры конфигурации модуля linstor. Настройка data-locality — размещение приложения возле данных. Storage-based fencing. Перенос приложения при проблемах с узлом.
 ---
 
+{% alert level="danger" %}
+Текущая версия модуля устарела и больше не поддерживается. Переключитесь на использование модуля [sds-drbd]().
```
Comment (z9r5) [inline modules/041-linstor/docs/USAGE_RU.md:7]:
```suggestion
Текущая версия модуля устарела и больше не поддерживается. Переключитесь на использование модуля [sds-drbd](https://deckhouse.ru/modules/sds-drbd/beta/).
```

**Review by kirkonru — APPROVED** [review]

**Pull request merged**